data/events.json

```diff
@@ -61,5 +61,21 @@
                 ]
             }
         ]
+    },
+    {
+        "date": "08-01-2017",
+        "items": [
+            {
+                "title": "Car bomb explosion in Baghdad",
+                "description": [
+                    "The Islamic State (IS) has claimed responsibility behind the blast.",
+                    "The bomb was set off in a vegetable market.",
+                    "11 people have been killed and dozens more injured."
```

Comment:
Not sure if message about reordering came through:

To me, the order makes more sense being:

- 11 killed
- vegetable market
- IS

This keeps the most important information at the top.